contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -108,6 +108,64 @@ END
 $BODY$
 LANGUAGE plpgsql STABLE STRICT PARALLEL SAFE;
 
+-- helper functions for XML VALUE(xpath)
+CREATE OR REPLACE FUNCTION sys.bbf_xmlvalue(xpath_pattern TEXT, datatype TEXT, xml_element ANYELEMENT)
```

Comment:
Do we have testcase for when all the input is NULL?

Reply:
yes, its there in xml_value-vu-verify test file